packages/widgets/src/components/metric/metric.widget.tsx

```diff
@@ -1,30 +1,29 @@
 import React, { FC } from 'react';
 import { MetricChart, MetricChartSettings } from '@keen.io/charts';
-import { Card, Badge } from '@keen.io/ui-core';
+import { Card } from '@keen.io/ui-core';
 
-import { Container, TagContainer } from './metric.widget.styles';
+import WidgetHeading from '../widget-heading.component';
+import { WidgetSettings } from '../../types';
 
-import { WidgetSettings, Tag } from '../../types';
+import { RelativeContainer } from './metric.styles';
 
 type Props = WidgetSettings & MetricChartSettings;
 
 /** Metric Chart widget integrated with other components */
-export const MetricChartWidget: FC<Props> = ({ card, tags, ...props }) => {
+export const MetricChartWidget: FC<Props> = ({
+  title,
+  subtitle,
+  card,
+  tags,
+  ...props
+}) => {
   return (
     <>
       <Card {...card} hideOverflow>
-        <Container>
-          {tags && tags.length > 0 && (
-            <TagContainer>
-              {tags.map(({ label, variant }: Tag) => (
-                <Badge key={label} variant={variant}>
-                  {label}
-                </Badge>
-              ))}
-            </TagContainer>
-          )}
-          <MetricChart {...props} />
-        </Container>
+        <RelativeContainer>
+          <WidgetHeading title={title} subtitle={subtitle} tags={tags} />
```

Comment:
IMO the metric chart plot should be **untouched** in this contribution. `<WidgetHeading />` should be positioned at widget level.